Error: Cannot find module 'node:path'

it seems that latest version is using node api from v16, older version of nodejs are breaking up

```
Error: The plugin /Users/pranaydutta/Desktop/projects/ice/loc-mobi/node_modules/typedoc-theme-hierarchy/dist/index.js could not be loaded.
Info: Error: Cannot find module 'node:path'
Require stack:
- /Users/pranaydutta/Desktop/projects/ice/loc-mobi/node_modules/typedoc-theme-hierarchy/dist/themes/OverrideTheme.js
- /Users/pranaydutta/Desktop/projects/ice/loc-mobi/node_modules/typedoc-theme-hierarchy/dist/index.js
- /usr/local/lib/node_modules/typedoc/dist/lib/utils/plugins.js
- /usr/local/lib/node_modules/typedoc/dist/lib/utils/index.js
- /usr/local/lib/node_modules/typedoc/dist/lib/models/reflections/project.js
- /usr/local/lib/node_modules/typedoc/dist/lib/models/reflections/index.js
- /usr/local/lib/node_modules/typedoc/dist/lib/models/index.js
- /usr/local/lib/node_modules/typedoc/dist/lib/converter/context.js
- /usr/local/lib/node_modules/typedoc/dist/lib/converter/index.js
- /usr/local/lib/node_modules/typedoc/dist/lib/application.js
- /usr/local/lib/node_modules/typedoc/dist/index.js
- /usr/local/lib/node_modules/typedoc/bin/typedoc
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:889:15)
    at Function.Module._load (internal/modules/cjs/loader.js:745:27)
    at Module.require (internal/modules/cjs/loader.js:961:19)
    at require (internal/modules/cjs/helpers.js:92:18)
    at Object.<anonymous> (/Users/pranaydutta/Desktop/projects/ice/loc-mobi/node_modules/typedoc-theme-hierarchy/dist/themes/OverrideTheme.js:18:37)
    at Module._compile (internal/modules/cjs/loader.js:1072:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1101:10)
    at Module.load (internal/modules/cjs/loader.js:937:32)
    at Function.Module._load (internal/modules/cjs/loader.js:778:12)
    at Module.require (internal/modules/cjs/loader.js:961:19)
```

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "typedoc-theme-hierarchy",
-  "version": "1.0.11",
+  "version": "1.0.13",
   "license": "ISC",
   "main": "dist/index.js",
   "repository": "https://github.com/DiFuks/typedoc-theme-hierarchy",
@@ -29,7 +29,8 @@
   },
   "scripts": {
     "build": "webpack && tsc --project tsconfig.build.json",
-    "lint": "tsc --noEmit && eslint \"./**/*.{ts,tsx,js}\""
+    "lint": "tsc --noEmit && eslint \"./**/*.{ts,tsx,js}\"",
+    "prepublishOnly": "yarn lint && yarn build"
   },
   "dependencies": {
     "fs-extra": "^10.0.0"

--- a/src/themes/OverrideTheme.tsx
+++ b/src/themes/OverrideTheme.tsx
@@ -6,7 +6,8 @@ import {
 } from 'typedoc';
 import { Renderer } from 'typedoc/dist/lib/output/renderer';
 import { copy } from 'fs-extra';
-import path from 'node:path';
+// eslint-disable-next-line unicorn/prefer-node-protocol
+import path from 'path';
 
 import { navigation } from '../partials/navigation';
 

